idea/testData/intentions/convertCollectionConstructorToFunction/keepLinkedHashSetCall.kt

```diff
@@ -0,0 +1,8 @@
+// WITH_RUNTIME
+// IS_APPLICABLE: false
+
+import java.util.LinkedHashSet
+
+fun foo() {
+    var list: LinkedHashSet<Int> = <caret>LinkedHashSet()
```

Comment:
why is this `IS_APPLICABLE: false`? It seems that a conversion should be applicable here: `LinkedHashSet` to `linkedSetOf()`.

Reply:
See above ^